lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/dva/DvaThirdPartyDocumentGateway.java

```diff
@@ -232,6 +240,8 @@ private DocumentCheckResult responseHandler(
                 throw new OAuthHttpResponseExceptionWithErrorBody(
                         HttpStatusCode.INTERNAL_SERVER_ERROR,
                         ErrorResponse.FAILED_TO_UNWRAP_DVA_RESPONSE);
+            } catch (NoSuchAlgorithmException e) {
+                throw new RuntimeException(e);
```

Comment:
I dont like that were burying and hiding this exception. It needs to returning either the exisiting exception or a custom one with a log. If things start to fail we want to be able to ID them asap

Reply:
Existing exception added.